android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java

```diff
@@ -183,11 +194,11 @@ public void progressLayoutAnimation(int tag, Map<String, Object> newStyle) {
     ViewManager viewManager = resolveViewManager(tag);
     ViewManager parentViewManager = resolveViewManager(parent.getId());
 
-    if (viewManager == null || parentViewManager == null) {
+    if (viewManager == null) {
```

Comment:
why do we remove that parentViewManager check? is it ok to send null parentViewManager to setnewProps now?

Reply:
During shared transition views are attached to our custom transition container which is just a native `View` without `ReactViewManager`. I added null-check to the method where `parentViewManager` is used, so I think that is safe.